README.md

```diff
@@ -522,37 +563,60 @@
     if (test)
       return false;
 
+    // acceptable (only for very simple statements and early function return)
+    if (test) return false;
+
     // good
     if (test) {
       return false;
     }
 
-    // good
-    function () { return false; }
+    // bad
+    function() { return false; }
 
     // good
-    function () {
+    function() {
       return false;
     }
+    ```
 
-    // acceptable (only for very simple statements and early function return)
-    if (test) return false;
+  - If you're using multi-line blocks with `if` and `else`, put `else` on the same line as your
+    `if` block's closing brace.
+
+    ```javascript
+    // bad
+    if (test) {
+      thing1();
+      thing2();
+    }
+    else {
+      thing3();
+    }
+
+    // good
+    if (test) {
+      thing1();
+      thing2();
+    } else {
+      thing3();
+    }
     ```
 
-    **[[⬆]](#TOC)**
+
+**[⬆ back to top](#table-of-contents)**
 
 
-## <a name='comments'>Comments</a>
+## Comments
 
-  - Use `/** ... */` for multiline comments. Include a description, specify types and values for all parameters and return values.
+  - Use `/** ... */` for multi-line comments. Include a description, specify types and values for all parameters and return values.
 
     ```javascript
     // bad
     // make() returns a new element
     // based on the passed in tag name
     //
-    // @param <String> tag
-    // @return <Element> element
+    // @param {String} tag
```

Comment:
This is to match the jsdoc's syntax, which I believe was the original intent.

http://usejsdoc.org/tags-param.html